src/components/form-helpers/form-helpers.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
The idea of this file is can reuse in all app?

Reply:
Yes, at least in signup, login and that kind of pages.